week2.ipynb

```diff
@@ -0,0 +1,288 @@
+{
```

Comment:
<p>@aeckland2696&nbsp;</p><p>Most of the unexpected behaviour and warnings that you encountered in this problem set and in the answer key stem from <code>geom_smooth</code>  trying to fit with a loess function by default (loess = Local Polynomial Regression Fitting). This is the default because it can always fit <em>something</em> but rarely what you want it to fit and often with a lot of warnings because your dataset might be too small for loess to actually work with its default parameters (i.e. the local polynomial fit is often underconstrained). You provided a formula here so I'm guessing you wanted it to use linear regression instead. So the key change here would be to add <code>method = "lm"</code>  to your <code>geom_smooth</code>  call:</p><p> </p><p><code>geom_smooth(<strong>method = "lm",</strong> formula = y ~ log(x)) +</code></p><p> </p><p>Note that this should also make the <code>suppressWarnings</code>  wrapper unnecessary as you no longer have an underconstrained fitting method.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-aeckland2696/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>